test/collectionUtils/ModifyExtensionTypeTransformer.test.ts

```diff
@@ -3,87 +3,40 @@ import EntityDefinition from "../../src/model/dataTypes/EntityDefinition";
 
     entityBuilder = new EntityDefinitionBuilder();
     otherEntity = entityBuilder.buildEntityDefinition();
 
-    valueMember = new MemberVariable(
-      typeType,
-      "value",
-      false,
-      undefined,
-      false,
-      [elementType, extensionType]
-    );
-
     entityBuilder.dataType = extensionType;
-    entityBuilder.imports = new EntityImports([elementType]);
-    entityBuilder.memberVariables.push(valueMember);
+    entityBuilder.imports = new EntityImports([]);
 
     extensionEntity = entityBuilder.buildEntityDefinition();
-    [member1, member2] = extensionEntity.memberVariables;
   });
 
   it("should leave non-Extension types alone", () => {
     const result = transformer.transform(otherEntity);
     expect(result).toBe(otherEntity);
-    expect(result.parentDataType).not.toBeNull();
+    expect(result.imports.dataTypes).toBeArrayOfSize(2);
+    expect(result.imports.dataTypes[0].normalizedName).toBe("memberTypeName1");
+    expect(result.imports.dataTypes[1].normalizedName).toBe("memberTypeName2");
   });
 
-  it("should remove the parent of Extension types", () => {
+  it("should add the IElement import to Extension types", () => {
     const result = transformer.transform(extensionEntity);
     expect(result).not.toBe(extensionEntity);
-    expect(result.parentDataType).toBeNull();
-  });
-
-  it("should remove the Element import from Extension type", () => {
-    expect(extensionEntity.imports.dataTypes).toStrictEqual([elementType]);
-    const result = transformer.transform(extensionEntity);
-    expect(result.imports.dataTypes).toBeArrayOfSize(0);
-  });
-
-  it("should remove and re-add the 'value' MemberVariable ", () => {
-    const result = transformer.transform(extensionEntity);
-    expect(result.memberVariables).toBeArrayOfSize(6);
-    expect(result.memberVariables[2].variableName).toBe("value");
-    expect(result.memberVariables[2].choiceTypes).toBeArrayOfSize(0);
-  });
-
-  it("should add a new id member to the Extension type", () => {
-    expect(extensionEntity.memberVariables).toStrictEqual([
-      member1,
-      member2,
-      valueMember,
-    ]);
-    const result = transformer.transform(extensionEntity);
-    expect(result.memberVariables).toBeArrayOfSize(6);
-    expect(result.memberVariables[3].variableName).toBe("id");
-    expect(result.memberVariables[3].dataType).toBe(SystemString);
-    expect(result.memberVariables[3].isArray).toBeFalse();
-    expect(result.memberVariables[4].variableName).toBe("extension");
-    expect(result.memberVariables[4].dataType).toBe(extensionType);
-    expect(result.memberVariables[4].isArray).toBeTrue();
-    expect(result.memberVariables[5].variableName).toBe("fhirId");
-    expect(result.memberVariables[5].dataType).toBe(SystemString);
-    expect(result.memberVariables[5].isArray).toBeFalse();
+    expect(result.imports.dataTypes).toBeArrayOfSize(1);
+    expect(result.imports.dataTypes[0].normalizedName).toBe("IElement");
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5923420)